.github/workflows/ci.yml

```
@@ -74,6 +78,14 @@ jobs:
       - name: Generate coverage summary
         if: always()
         run: |
```

actionlint (1.7.8)

property "job-start" is not defined in object type {}

(expression)